src/clr-angular/wizard/_properties.wizard.scss

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('wizard.properties') {
+  @if $clr-use-custom-properties == true {
+    :root {
+      --clr-wizard-main-bgcolor: var(--clr-color-neutral-0);
+      --clr-wizard-sidenav-bgcolor: var(--clr-color-neutral-50);
+      --clr-wizard-main-text-color: var(--clr-color-on-neutral-0);
+      --clr-wizard-sidenav-text: var(--clr-color-neutral-900);
+      --clr-wizard-sidenav-text--active: var(--clr-global-on-selection-color);
+      --clr-wizard-title-text: var(--clr-color-neutral-1000);
+      --clr-wizard-stepnav-border-size: #{$clr_baselineRem_4px};
+      --clr-wizard-stepnav-border-color: var(--clr-color-neutral-200);
+      --clr-wizard-stepnav-border-color--active: var(--clr-color-success-400);
+      --clr-wizard-stepnav-active-bgcolor: var(--clr-global-selection-color);
+      --clr-wizard-header-action-color: var(--clr-color-neutral-600);
+      --clr-wizard-header-action-color--hovered: var(--clr-color-neutral-1000);
+      --clr-wizard-border-radius: var(--clr-global-borderradius);
+      --clr-wizard-stepnav-active-border-radius: var(--clr-wizard-border-radius);
+
+      --clr-wizard-step-nav-border-color: hsl(0, 0%, 89%);
+      --clr-wizard-box-shadow-color: rgba(0, 0, 0, 0.2);
```

Comment:
Are these color values just specific to wizard?

Reply:
Yes. :-(